modules/admin_manual/pages/release_notes.adoc

```diff
@@ -24,6 +25,151 @@
+TIP: If your ownCloud deployment is based on the official Docker images or the Univention appliance, these changes have already been applied for you.
+
+=== Media Viewer replaces Gallery and Video Player
+
+Recently the https://marketplace.owncloud.com/apps/files_mediaviewer[Media Viewer] app has been released. The Media Viewer is the next generation image and video file viewer for ownCloud.
+It provides a foundation based on new technologies and officially supersedes the former `gallery` and `files_videoplayer`.
+ownCloud Server 10.3 does not bundle `gallery` and `files_videoplayer` anymore. Instead it bundles `files_mediaviewer`. With this, support and maintenance for `gallery` and `files_videoplayer` is discontinued. More details on the Media Viewer can be found in the https://owncloud.org/news/good-bye-gallery-say-hi-to-the-new-media-viewer/[release blog post].
+
+* For a clean transition to Media Viewer it is necessary to **disable both deprecated apps prior to the upgrade** using either the admin "Apps" panel in the web interface or via `occ` (e.g., `occ app:disable gallery files_videoplayer`). 
+* After the upgrade, enable the Media Viewer app via the admin panel or `occ app:enable files_mediaviewer`.
+* It is not recommended to continue with the deprecated apps. However, if you want to do so, you can copy over the `files_videoplayer` directory from the `apps/` folder of the previous ownCloud Server directory and obtain `gallery` from the https://marketplace.owncloud.com/apps/gallery[ownCloud Marketplace].
+
+TIP: Please do not enable `gallery`/`files_videoplayer` and `files_mediaviewer` simultaneously as these apps are mutually exclusive.
+
+For more information on the Media Viewer, visit the https://doc.owncloud.com/server/admin_manual/installation/apps/mediaviewer/[ownCloud Documentation].
+
+=== OAuth2 and session handling improvements
+
+Server 10.3 comes with improvements for session handling with Redis to cope for issues encountered around duplicate session tokens making the ownCloud Clients loose their OAuth2 authorization from time to time and forcing users to reauthenticate. The session handling stability has been generally improved and support for Redis Session Locking has been introduced. If Redis is used, it is necessary to enable Session Locking to ensure that the mentioned issues will not occur anymore.
+
+* It is recommended to use Redis Session Locking (minimum required version for `php-redis` is `4.1.0`)
+* Enable Redis Session Locking by setting `redis.session.locking_enabled = 1` in `php.ini`
+
+TIP: Please note that Redis Session Locking is not supported in clustered Redis environments.
+
+TIP: If you do not use Redis at all, you do not have to apply changes.
+
+TIP: If your ownCloud deployment is based on the official Docker images or the Univention appliance, these changes have already been applied for you.
```

Comment:
hopefully, not tested yet.

Reply:
This isn't guaranteed?

Reply:
Ubuntu 18.04 has php-redis v3
Docker containers have Ubuntu 18.04

Ubuntu 20.04 not ready.